libsrc/telestrat/textcolor.s

```diff
@@ -0,0 +1,27 @@
+; 2019-07-02, Jede (jede@oric.org)
+;
+
+        .export         _textcolor
+        .import         CHARCOLOR
+        .import         CHARCOLOR_CHANGE
+        .include        "telestrat.inc"
+
+.proc _textcolor
+    cmp     CHARCOLOR       ; Do we set the same color? If we don't detect it, we loose one char on the screen for each textcolor call with the same color
+    bne     out             ; yes
+    
+    ldy     #$00
+    sty     CHARCOLOR_CHANGE
+    
+    lda     CHARCOLOR       ; Return last color
```

Comment:
The two lines above are not needed (similar to `bgcolor.s`).

Reply:
Corrected